modules/stanford_earth_capx/stanford_earth_capx.module

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @file
+ * File description.
+ *
+ * Long description.
+ */
+
+use Drupal\migrate\Plugin\MigrationInterface;
+use Drupal\migrate\Row;
+use Drupal\migrate\Plugin\MigrateSourceInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\stanford_earth_capx\EarthCapxInfo;
+
+/**
+ * Implements hook_entity_delete().
+ */
+function stanford_earth_capx_entity_delete(EntityInterface $entity) {
+
+  // only do this for user content
+  if ($entity->bundle() !== 'user') {
```

Comment:
Do we need this check? There are users that are not from CAP that could be deleted and they would not have a record.

Reply:
My understanding is that this will get called for any entity that is deleted, so if there is a node id that happens to be the same as a user id,  the information for that user would be deleted when the node is deleted. I think the bundle check will prevent incorrect or at lease unneeded processing.

Reply:
Ah good point and yes you are correct. Disregard my comment above.